lib/v1/index.js

```diff
@@ -1,14 +1,13 @@
 const { Datatypes, errors, extractDomain } = require('./utils');
 const Config = require('./config');
 const { Crypto, Http, Forms } = require('./core');
-const extractDomainFromUrl = require('./utils/domain');
 
 class EvervaultClient {
-  constructor(teamId) {
+  constructor(teamId, configUrls) {
```

Comment:
This should have a more generic name, to allow us to extend it. 
```suggestion
  constructor(teamId, config) {
```